modules/renter.go

```diff
@@ -52,6 +52,7 @@ type FileInfo struct {
 	Filesize       uint64            `json:"filesize"`
 	Available      bool              `json:"available"`
 	Renewing       bool              `json:"renewing"`
+	Redundancy     float64           `json:"redundancy"`
 	UploadProgress float64           `json:"uploadprogress"`
```

Comment:
It might be confusing to have each number represent something different. I think I'd prefer to use `Redundancy` alone, as it gives the user a picture that is more relevant to them.